src/wasm.h

```diff
@@ -656,6 +656,7 @@ class Import {
   ExternalKind kind;
   Name functionType; // for Function imports
   Type globalType; // for Global imports
+  bool mutable_; // for Global imports
```

Comment:
This feels dirty but seemed like a simple solution without deciding to completely rethink about imports/globals are represented